modules/ROOT/partials/configuration/dialog_persistent.adoc

```diff
@@ -0,0 +1,109 @@
+
+If an end-user gives another part of the {productname} editor focus (for example, the document editor, or a menu), the inline dialog will stay open.
+
+Setting the property to `+true+` does not over-ride normal mechanisms for closing a dialog, such as clicking the _Close_ button or pressing the *Esc* key.
+
+The property has a default value of `+false+`.
+
+NOTE: If an inline dialog is open and focus is switched away from the {productname} editor entirely (by, for example, the end-user switching focus to an available host-browser UI element), an open inline dialog will remain open regardless of the `+persistent+` property’s value. This was the behavior prior to this property being available and continues to be the behavior.
+
+*Type:* `+Boolean+`
+
+*Possible values:* `+true+`, `+false+`
+
+*Default value:* `false`
+
+=== Example: using `persistent`
+
+[source,js]
+----
+tinymce.activeEditor.windowManager.open({
+  title: 'Dialog Title', // The dialog's title - displayed in the dialog header
+  body: {
+    type: 'panel', // The root body type - a Panel or TabPanel
+    items: [ // A list of panel components
+      {
+        type: 'htmlpanel', // an HTML panel component
+        html: 'Panel content goes here.'
+      }
+    ]
+  },
+  buttons: [ // A list of footer buttons
+    {
+      type: 'submit',
+      text: 'OK'
+    }
+  ]},
+  { inline: 'toolbar', persistent: true }
+});
+----
+
+=== Example: using `focus` to refocus an open but de-focused dialog
+
+Because a dialog with a `{ persistent: true }` property does not close when the dialog loses focus, it is possible for an end-user to open a further instance of the dialog.
+
+To prevent this
+
+* add a check that the dialog is already open; and
+
+* use the xref:apis/tinymce.editor/#focus[`focus`] method in the xref:apis/tinymce.windowmanager/#return-value[`DialogInstanceAPI`] to transfer focus to the open dialog rather than creating a new instance of the same dialog.
+
+
+[source,js]
+----
+let dialogOpened = false;
+let dialogInstanceApi = null;
+
+const openDialog = () => {
+  dialogOpened = true;
+  dialogInstanceApi = editor.windowManager.open({
+    title: 'Test',
+    body: {
+      type: 'panel',
+      items: [{
+        type: 'bar',
+        items: [{
+            type: 'input',
+            name: 'name',
+            placeholder: 'Name',
+            maximized: true
+          }
+        ]
+      }]
+    },
+    onClose: () => {
+      dialogOpened = false;
+    },
+    buttons: [{
+      type: 'cancel',
+      name: 'cancel',
+      text: 'Cancel'
+    }],
+    initialData: {
+      name: ''
+    }
+  }, {
+    inline: 'bottom',
+    persistent: true
+  });
+};
+
+const focusDialog = () => {
+  dialogInstanceApi.focus('name');
+}
+
+editor.ui.registry.addButton('example', {
+  type: 'button',
+  text: 'example',
+  tooltip: 'example',
+  onAction: () => {
+    dialogOpened ? focusDialog() : openDialog();
+  }
+});
+----
```

Comment:
For demo purposes here is a fiddle created by Shan [Persistent dialog](https://fiddle.tiny.cloud/DBiaab)